bool stop_transmission
---
bool transmission_stopped